challenges/fifo_animal_shelter/node.py

```diff
@@ -0,0 +1,9 @@
+class Node(object):
+    def __init__(self, animalName=None, animalKind=None, pointer=None):
+        self.animalName = animalName
+        self.animalKind = animalKind
```

Comment:
Instead of storing the animal name and type on the Node, you should create Cat and Dog classes that have that information on them. Then store the Cat and Dog objects as the value of the Node.